File: Program/Tier2/src/main/java/via/sep3/group11/tier2/shared/DTOs/HostDTO.java
package via.sep3.group11.tier2.shared.DTOs;

import via.sep3.group11.tier2.shared.domain.Host;

public class HostDTO {
    private Host host;
    private String errorMessage;

    public HostDTO(Host host, String errorMessage) {
        this.host = host;
        this.errorMessage = errorMessage;
    }

    public Host getHost() {
        return host;
    }

    public String getErrorMessage() {
        return errorMessage;
    }
}

File: Program/Tier1/Shared/DTOs/HostDTO.cs
using Shared.Domain;

namespace Shared.DTOs;

public class HostDTO
{
    public Host Host { get; }
    public String ErrorMessage { get; }

    public HostDTO(Host host, string errorMessage)
    {
        this.Host = host;
        this.ErrorMessage = errorMessage;
    }
}